text-generation-inference/docker/Dockerfile

```diff
@@ -1,6 +1,7 @@
-# Fetch and extract the TGI sources (TGI_VERSION is mandatory)
+# Fetch and extract the TGI sources
 FROM alpine AS tgi
-ARG TGI_VERSION
+# TGI version 2.4.1 by default
+ARG TGI_VERSION=690702b1ce9a27ce5bdf2a9dd3a80277ecea12cd 
```

Comment:
why not 2.4.1?

Reply:
Because we used the sha in the makefile so i aligned with that but let me change it then

Reply:
it is an error on my side! I thought it was v2.4.1 by now...